src/three/plugins/images/sources/XYZImageSource.js

```diff
@@ -49,3 +49,57 @@ export class XYZImageSource extends TiledImageSource {
 	}
 
 }
+
+// Bing Maps Tile System
+// https://learn.microsoft.com/en-us/bingmaps/articles/bing-maps-tile-system
```

Comment:
From this link - it looks like the Bing maps are going to be retired in a few years. Are there any other tiling schemes that use this quad key system?